web_ui/frontend/components/Namespace/InformationDropdown.tsx

```diff
@@ -0,0 +1,74 @@
+import {Box, Tooltip, Collapse, Grid, Typography} from "@mui/material";
+import React from "react";
+import {NamespaceAdminMetadata} from "./index.d";
+
+const InformationSpan = ({name, value}: {name: string, value: string}) => {
+    return (
+        <Tooltip title={name} placement={"right"}>
+            <Box sx={{
+                "&:nth-of-type(odd)": {
+                    bgcolor: "#ececec",
+                    p: "4px 6px",
+                    m: "0px -5px",
+                    borderRadius: "4px"
+                },
+                "&:nth-of-type(even)": {
+                    p: "4px 6px",
+                    m: "0px -5px"
+                },
+                display: "flex",
+                justifyContent: "space-between",
+            }}>
+                <Typography variant={"body2"} sx={{fontWeight: 500,  display: "inline"}}>{name}</Typography>
+                <Typography variant={"body2"} sx={{display: "inline"}}>{value}</Typography>
+            </Box>
+        </Tooltip>
+    )
+}
+
+interface InformationDropdownProps {
+    adminMetadata: NamespaceAdminMetadata;
+    transition: boolean;
+    parentRef?: React.RefObject<HTMLDivElement>;
+}
+
+const InformationDropdown = ({adminMetadata, transition, parentRef} : InformationDropdownProps) => {
+
+    const information = [
+        {name: "User ID", value: adminMetadata.user_id},
+        {name: "Description", value: adminMetadata.description},
+        {name: "Site Name", value: adminMetadata.site_name},
+        {name: "Institution", value: adminMetadata.institution},
+        {name: "Security Contact User ID", value: adminMetadata.security_contact_user_id},
+        {name: "Status", value: adminMetadata.status},
+        {name: "Approver ID", value: adminMetadata.approver_id.toString()},
+        {name: "Approved At", value: adminMetadata.approved_at},
```

Comment:
This strikes me as more of a server side issue. I will add a check, but if the value isn't set it should return null.

Reply:
For go, the default value for date is `0001-01-01T00:00:00Z` , but I can change how I serialize the date fields and return empty string if it's in default value. Does that sound good? I don't think I can pass a `null` or `undefined` value tho because date was stored as a value not a pointer in the Go struct

Reply:
Okay, not a big deal. I will assume `0001-01-01T00:00:00Z` is the null value in the future for dates.